Tasks/AzureDevOpsTestPlanV0/getAutomatedTests.ts

```diff
@@ -0,0 +1,105 @@
+﻿import { Console } from "console";
+import tl = require('azure-pipelines-task-lib/task');
+import apim = require('azure-devops-node-api');
+import { WorkItemDetails, TestCase } from 'azure-devops-node-api/interfaces/TestPlanInterfaces';
+import { PagedList } from 'azure-devops-node-api/interfaces/common/VSSInterfaces';
+import TestPlanInterfaces = require('azure-devops-node-api/interfaces/TestPlanInterfaces');
+import VSSInterfaces = require('azure-devops-node-api/interfaces/common/VSSInterfaces');
+
+
+export interface TestPlanData {
+    testPointIds: number[];
+    listOfFQNOfTestCases: string[];
+}
+
+ const AutomatedTestName = "Microsoft.VSTS.TCM.AutomatedTestName";
+ const AutomatedTestStorage = "Microsoft.VSTS.TCM.AutomatedTestStorage";
+export async function getAutomatedTestData(testPlanId: number, testSuiteIds: number[], testConfigurationId: number): Promise<TestPlanData> {
+    
+    const testPlanData: TestPlanData = { testPointIds: [], listOfFQNOfTestCases: [] };
+    let token = null;
+
+    if (testPlanId == 0 || testConfigurationId == 0) {
+        return testPlanData;
+    }
+
+    for (const testSuiteId of testSuiteIds) {
+        if (testSuiteId === 0) {
+            continue;
+        }
+
+        let testCasesData: TestCase[] = [];
+
+        do {
+            try {
+                let testCasesResponse = await APICall1(testPlanId, testSuiteId, testConfigurationId.toString(), token);
+
+                token = testCasesResponse.continuationToken;
```

Comment:
Should we move this token assignment to above line 35 , since we are passing this token to the new API call as well?

Reply:
Initially continuation token will be null, then it will set on based on what is returned from API.
Hence, we cannot move it from here as it depends on API response